app/views/_init-count.html

```diff
@@ -0,0 +1,11 @@
+<!--
+  Expects the following variables:
+  podTemplate
+ -->
+
+<span ng-if="pod.spec.initContainers.length" class="text-muted small">
+  <ng-pluralize count="pod.spec.initContainers.length"
+                when="{'1': '&nbsp;Init container','other': '&nbsp;{} Init containers'}">
+  </ng-pluralize>
+  <a ng-href="{{pod | navigateToTabURL:'details'}}">View Details</a>
```

Comment:
@spadgett need some input about ```details``` not being the right tab. 

Can place a check/condition that alters the tab between ```details``` and ```configuration```, but unsure of what property to check against?